data/json/legacy_artifact_active.json

```diff
@@ -70,6 +70,19 @@
     "max_damage": -15,
     "message": "You're wracked with pain!"
   },
+  {
+    "type": "SPELL",
+    "id": "AEA_TELEPORT",
+    "name": "Artifact Teleport",
+    "description": "Randomly teleports you a short distance.",
+    "effect": "teleport_random",
+    "valid_targets": [ "self" ],
+    "flags": [ "NO_HANDS", "SILENT", "NO_LEGS", "UNSAFE_TELEPORT" ],
+    "min_range": 2,
+    "max_range": 12,
+    "max_level": 1,
```

Comment:
i've been leaving level out of the AEA spells. they'll probably eventually be obsoleted, in favor of some other kind of procedurally generated things.